rhel7/profiles/ospp.profile

```diff
@@ -83,104 +90,454 @@ selections:
     - sysctl_net_ipv4_ip_forward
     - sysctl_net_ipv4_tcp_syncookies
 
-    # Module blacklist
-    - kernel_module_usb-storage_disabled
-    - kernel_module_cramfs_disabled
-    - kernel_module_bluetooth_disabled
-    - kernel_module_dccp_disabled
-    - kernel_module_sctp_disabled
+    ### systemd
+    - disable_ctrlaltdel_reboot
+    - disable_ctrlaltdel_burstaction
+    - service_debug-shell_disabled
+    - service_kdump_disabled
+    - service_autofs_disabled
 
-    ### Audit
-    - service_auditd_enabled
+    ### umask
+    #- var_accounts_user_umask=027
+    - accounts_umask_etc_profile
+    - accounts_umask_etc_bashrc
+    - accounts_umask_etc_csh_cshrc
+    #- accounts_umask_etc_login_defs
 
-    # Rules.d
-    - directory_access_var_log_audit
+    ### Software update
+    - ensure_redhat_gpgkey_installed
+    - ensure_gpgcheck_globally_activated
+    - ensure_gpgcheck_local_packages
+    - ensure_gpgcheck_never_disabled
 
-    # Configuration (all are defaults)
+    ### Passwords
+    - var_password_pam_difok=4
+    - accounts_password_pam_difok
+    - var_password_pam_maxrepeat=3
+    - accounts_password_pam_maxrepeat
+    - var_password_pam_maxclassrepeat=4
+    - accounts_password_pam_maxclassrepeat
+
+    ### Kernel Config
+    ## Boot prompt
+    - package_dracut-fips_installed
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
+    #- grub2_pti_argument
+
+    ## Security Settings
+    - sysctl_kernel_kptr_restrict
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_kexec_load_disabled
+    - sysctl_kernel_yama_ptrace_scope
+    #- sysctl_kernel_perf_event_paranoid
+    #- sysctl_user_max_user_namespaces
+    #- sysctl_kernel_unprivileged_bpf_disabled
+    #- sysctl_net_core_bpf_jit_harden
+
+    ## File System Settings
+    - sysctl_fs_protected_hardlinks
+    - sysctl_fs_protected_symlinks
+
+    ### Audit
+    - service_auditd_enabled
     - var_auditd_flush=incremental_async
     - auditd_data_retention_flush
+    #- auditd_local_events
+    #- auditd_write_logs
+    #- auditd_log_format
+    #- auditd_freq
+    #- auditd_name_format
 
-    # TODO: check that auditd has rules configured according to
-    #       Configuration Annex for OSPP 4.2.1
+    ### Misc Audit Configuration
+    ### (not required in OSPP)
+    #- audit_rules_session_events
+    #- audit_rules_mac_modification
 
-    ### Services
-
-    # sshd
-    - sshd_disable_root_login
-    - sshd_enable_strictmodes
-    - disable_host_auth
-    - sshd_disable_empty_passwords
-    - sshd_disable_kerb_auth
-    - sshd_disable_gssapi_auth
-    - sshd_idle_timeout_value=10_minutes
-    - sshd_set_keepalive
-    - sshd_enable_warning_banner
-    - sshd_disable_rhosts_rsa
-    - sshd_use_approved_ciphers
-    - sshd_use_approved_macs
+    ### Module Blacklist
+    #- kernel_module_usb-storage_disabled
```

Comment:
Ahoj Shawn. This is one rule that you missed - it was enabled before. Can you uncomment it?

Reply:
Done, and thanks for the rule review. A surprising amount of selections to track.